arcgis-ios-sdk-samples/Display information/Add graphics with symbols/GraphicsWithSymbolsViewController.swift

```diff
@@ -72,160 +72,160 @@ class GraphicsWithSymbolsViewController: UIViewController {
         let buoyGraphic4 = AGSGraphic(geometry: buoy4Loc, symbol: buoyMarker, attributes: nil)
         
         //add the graphics to the graphics overlay
-        graphicsOverlay.graphics.addObjectsFromArray([buoyGraphic1, buoyGraphic2, buoyGraphic3, buoyGraphic4])
+        graphicsOverlay.graphics.addObjects(from: [buoyGraphic1, buoyGraphic2, buoyGraphic3, buoyGraphic4])
     }
     
-    private func addText(graphicsOverlay:AGSGraphicsOverlay) {
+    fileprivate func addText(_ graphicsOverlay:AGSGraphicsOverlay) {
```

Comment:
similar to other comments above about having argument name to clarify call site usage